src/google/protobuf/io/test_zero_copy_stream_test.cc

```diff
@@ -7,15 +7,15 @@
 
 #include "google/protobuf/io/test_zero_copy_stream.h"
 
+#include <gmock/gmock.h>
```

Comment:
Note: these header reorderings will cause merge conflicts and make our import super painful, try to avoid them if you can :)